libs/datamodel/core/src/transform/ast_to_dml/datasource_loader.rs

```diff
@@ -123,7 +123,7 @@ impl DatasourceLoader {
             }
         };
 
-        let url = match url_arg.as_str_from_env() {
+        let url = match url_arg.value.try_into() {
```

Comment:
I'd like the `try_from` here for clarity.

Reply:
done